### ISSUE TITLE
ES_HEAP_SIZE not working on v5

### ISSUE THREAD

**Issue by selman** [spujadas/elk-docker#201]
`/opt/elasticsearch/bin/elasticsearch` is firstly reading `jvm.options` from `$ES_HOME` not  from    `$CONF_DIR`. tested on `v5` branch.

```sh
if [ -z "$ES_JVM_OPTIONS" ]; then
    for jvm_options in "$ES_HOME"/config/jvm.options \
                      /etc/elasticsearch/jvm.options; do
        if [ -r "$jvm_options" ]; then
            ES_JVM_OPTIONS=$jvm_options
            break
        fi
    done
fi
```

**Comment by spujadas** [comment]
Thanks for reporting this, looks like this env var needs to be moved to `/etc/elasticsearch/jvm.options`.
(Unfortunately had a disk drive crash on my dev workstation so need to get a new one and won't get around to fixing this issue until then.)

**Comment by spujadas** [comment]
fixed by #203
